src/Framework/Framework/Resources/Scripts/knockout-latest.debug.js

```diff
@@ -5117,6 +5117,9 @@ ko.bindingHandlers['selectedOptions'] = {
                 previousScrollTop = element.scrollTop;
 
             if (newValue && typeof newValue.length == "number") {
+                // touch all array elements
```

Comment:
```suggestion
                // touch all array elements, because the ko.utils.arrayIndexOf bellow unwraps everything with ko.dependencyDetection.ignore
```